fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowWarningErrorsCommand.java

```diff
@@ -0,0 +1,68 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.plans.commands;
+
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.nereids.trees.plans.PlanType;
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSet;
+import org.apache.doris.qe.ShowResultSetMetaData;
+import org.apache.doris.qe.StmtExecutor;
+
+import com.google.common.collect.Lists;
+
+import java.util.List;
+
+/**
+ * Represents the command for SHOW WARNINGS or SHOW ERRORS.
+ */
+public class ShowWarningErrorsCommand extends ShowCommand {
+    private static final ShowResultSetMetaData META_DATA =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("Level", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("Code", ScalarType.createVarchar(10)))
+                    .addColumn(new Column("Message", ScalarType.createVarchar(100)))
+                    .build();
+
+    private final boolean isWarning;
+    private final long limit;
+
+    public ShowWarningErrorsCommand(boolean isWarning, long limit) {
+        super(PlanType.SHOW_WARNING_ERRORS_COMMAND);
+        this.isWarning = isWarning;
+        this.limit = limit;
+    }
+
+    @Override
+    public ShowResultSet doRun(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        List<List<String>> rowSet = Lists.newArrayList();
+        // Only success
+        return new ShowResultSet(this.getMetaData(), rowSet);
```

Comment:
we can directly use META_DATA directly no need function.

Reply:
done